ethereumetl/streaming/item_exporter_creator.py

```diff
@@ -24,25 +24,27 @@
 from blockchainetl.jobs.exporters.multi_item_exporter import MultiItemExporter
 
 DEFAULT_KAFKA_TOPIC_MAPPING = {
-    'block': 'blocks',
-    'transaction': 'transactions',
-    'log': 'logs',
-    'token_transfer': 'token_transfers',
+    'block': 'ethereum_blocks',
+    'transaction': 'ethereum_transactions',
+    'log': 'ethereum_logs',
+    'token_transfer': 'ethereum_token_transfers',
+    # todo(shashank): migrate token_transfers_v2 to new topic
     'token_transfer_v2': 'token_transfers_v2',
```

Comment:
handling this might be tricky as kafka does not allow renames:

- we could use `mirrormaker` to copy old data and then write to newer topic

or

- write to multiple topics and switch the consumer from the appropriate block_number

Reply:
standardizing the topics might be worth in the long run